docs/TELEMETRY_SPECIFICATION.md

```
@@ -1165,3 +1177,87 @@ telemetry:
+对接公司 Sentry 时按以下步骤配置和验证。
+
+**配置注意**：`zstack.properties` 中等号两边**不要加空格**（`key=value`），否则 key 带空格读不到；Sentry 测试时建议 `Telemetry.exporters=otlp,sentry`、`Telemetry.sentryTracesSampleRate=1.0`（或通过该配置项设置）。使用 Sentry 时需在 **Sentry.init()** 中设置 **tracesSampleRate**，否则 Sentry 可能不接收 OTel 发去的 transaction，Performance/Traces 可能看不到数据；ZStack 已通过 `Telemetry.sentryTracesSampleRate` 传入，默认 1.0。
+
+### B.1 前置条件
+
+- 已拿到 Sentry 项目的 **DSN**（形如 `https://xxx@xxx.ingest.sentry.io/xxx`）。
+- 管理节点 classpath 中包含 `sentry`、`sentry-opentelemetry-bootstrap`、`sentry-opentelemetry-core`（根 `pom.xml` 的 dependencyManagement 中已声明，build 打包时会带入 WEB-INF/lib）。
+
+### B.2 配置步骤
+
+**1. 配置 DSN**
```

markdownlint-cli2 (0.20.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)